zc_events/client.py

```diff
@@ -260,7 +260,9 @@ def delete_no_wait(self, key, data={}, headers={}):
             key, data, headers, False, 'delete_no_wait'
         )
 
-    def emit_microservice_message(self, exchange, routing_key, event_type, priority=0, *args, **kwargs):
+    def emit_microservice_message(  # pylint: disable=keyword-arg-before-vararg
+        self, exchange, routing_key, event_type, priority=0, *args, **kwargs
+    ):
```

Comment:
pylint doesnt like having default value keyword arguments before `*args, **kwargs` but I don't want to refactor the signature at this time.